authui/src/authflowv2/components/otp-input.css

```diff
@@ -80,6 +80,7 @@
     @apply items-stretch;
     @apply gap-x-[var(--otp-input\_\_spacing)];
     @apply pointer-events-none;
+    @apply justify-center;
```

Comment:
Revert this or you provide a reason for this change.

Reply:
Don't observe difference after disabling, removing this line first.